docs/troubleshooting/yellow-red-loop.md

```diff
@@ -6,7 +6,7 @@ Let's divide this page up into two parts. The first part deals with individual r
 
 So you are here for help with getting your Loop green again? It's red and not working? Wondering what is happening? You can click on the red loop icon to read an error message that you may or may not understand. If you understand it, great...that should help you fix the problem. If you don't understand it, then there are two options to fix the red loop; the first is quick and easy, the second is more in-depth. Let's look at both.
 
-**OmniPod Users**: Do not quickly decide to pull a pod that isn't responding and has a red loop. Usually the issue is that the RileyLink (or other radio-link) needs a reset as described below in Option 1, rather than the pod needing to be changed. If you don't fix the radio-link's problem first, the problem may carry forward into a failed pod-pairing attempt because the radio-link still isn't working. So, please try getting the radio-link working properly before deciding to ditch a pod because of a red loop.
+**OmniPod Users**: Do not quickly decide to pull a pod that isn't responding and has a red loop. Usually the issue is that the RileyLink compatible device needs a reset as described below in Option 1, rather than the pod needing to be changed. If you don't fix the device's problem first, the problem may carry forward into a failed pod-pairing attempt because it still isn't working. So, please try getting the device working properly before deciding to ditch a pod because of a red loop.
```

Comment:
I'd use "Omnipod" instead of "OmniPod" in Loop for consistency.

I'd suggest changing "before deciding to ditch a pod because of a red loop" or "before deciding to ditch a pod because of a failed pod setup or a red loop" (I've never ever had to ditch a pod because of a red loop in 2-1/2 years of use).